src/components/graph/menus/dynamic-simulation/event-modification-scenario-editor.tsx

```diff
@@ -75,14 +73,14 @@ const EventModificationScenarioEditor = () => {
         | {
               eventType?: EventType;
               equipmentId: string;
-              equipmentType: string;
+              equipmentType: keyof typeof EQUIPMENT_TYPE_LABEL_KEYS;
```

Comment:
Probably, but there is more search to do here, I think there is conflicts in some definitions...

![image](https://github.com/user-attachments/assets/426e0566-8025-4de6-9fe7-0baba42ecf0e)
>TS7053: Element implicitly has an any type because expression of type EQUIPMENT_TYPES can't be used to index type
>```typescript
>{
>     LINE: string;
>     TWO_WINDINGS_TRANSFORMER: string;
>     THREE_WINDINGS_TRANSFORMER: string;
>     HVDC_LINE: string;
>     LOAD: string;
>     GENERATOR: string;
>     BUS: string;
>}
>```
>Property '[EQUIPMENT_TYPES.SUBSTATION]' does not exist on type '{ LINE: string; TWO_WINDINGS_TRANSFORMER: string; THREE_WINDINGS_TRANSFORMER: string; HVDC_LINE: string; LOAD: string; GENERATOR: string; BUS: string; }'.